addons/binding/org.openhab.binding.boschindego/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,30 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: BoschIndego Binding
+Bundle-SymbolicName: org.openhab.binding.boschindego;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .,
+ lib/bosch-indego-controller-lib-0.8.jar,
+ lib/httpclient-4.5.2.jar,
+ lib/httpcore-4.4.4.jar,
+ lib/commons-codec-1.9.jar,
```

Comment:
not sure if I asked that on my initial review, but do you really need to bundle commons-codec and commons-logging? codec 1.6 is part of openHAB already and commons-logging should not be necessary as we have slf4j-api in place.